src/transformers/models/albert/modeling_albert.py

```diff
@@ -874,6 +874,10 @@ def forward(self, hidden_states):
 
         return prediction_scores
 
+    def _retie_weights(self):
+        # To re-tie those two weights if they get disconnected (on TPU)
+        self.bias = self.decoder.bias
```

Comment:
Do we still need the lines in the init of `AlbertMLMHead`:
```
        # Need a link between the two variables so that the bias is correctly resized with `resize_token_embeddings`
        self.decoder.bias = self.bias
```
now that this is taken care of by the `tie_weights` method?